lib/netflix2.js

```diff
@@ -56,6 +56,44 @@ Netflix.prototype.login = function (credentials, callback) {
   }
 }
 
+/**
+ * Browse movies, to simply get all films use Category ID 34399
+ *
+ * @param genreId The Netflix Category ID, Like https://www.netflix.com/browse/genre/34399
+ * @param page The content is paged, this is the page number.
+ * @param perPage How many items do you want per page?
+ * @param callback When the request is finished.
```

Comment:
```suggestion
 * @param callback Function to be called when the request is finished.
```

Reply:
You should probably also explain what paramaters the callback function is called with. That would answer my main question about this function: what information does it provide you with?